HSLdevcom/jore-map-ui
# 375 filter network at sidebar and when creating new route path

closes #375 

Redraw network layers when selectedTransitTypes change.

This implementation will be refactored a bit at card #400

How to test:

network filtering
1) open some route to the routeList
2) toggle network checkboxes (links and linkPoints)

new routePath filtering
1) open tram to route list
2) go to trams routePathView
3) click "luo uusi reitin suunta käyttäen tätä pohjana"
4) map should be filtered
